### PR TITLE
Add evolution land grant 

### PR DESCRIPTION
**Pull request by WoeOm**

### CODE REVIEW

**Comment by 1022ranji** [comment]
Could you please supply a test network for us to test the functions of the beta version?

**Comment by WoeOm** [comment]
> Could you please supply a test network for us to test the functions of the beta version?

Preview Link - https://www.evolution.land.l2me.com/land/3

**Comment by gravitiny** [comment]
Here is the test version https://www.evolution.land.l2me.com/land/3

**Comment by 1022ranji** [comment]
Alright, we'll test and give the feedback as soon as possible.

File: grant/projects/evolutionland.md [hunk 1]
```diff
@@ -0,0 +1,118 @@
...
+
+Evolution Land integrates various gameplays, including management, collection, education, gene, battle, auction, and game, which will continuously be evolving.
+
+**One Dapp One Community Metaverse**
+10+ projects have announced plans to build communities in Evolution Land to present project information and give back rewards to the community fans.
+
+### **Why is this project is good for the Crab ecosystem:**
+
+1. Evolution Land is the first build GameFi project on Crab Smart Chain. Evolution Land has been built on four network - Ethereum, Tron, HECO, and Polygon. Each of them have at least 200,000+ $RING TVL. It will definately bring more TVL on Crab Smart Chain.
+2. Evolution Land have 1000+ DAU from HECO, Ethereum, Tron, and Polygon. And will build on BSC and Moonriver in the future. It could introduce more users and assets from other chains.
```
Comment (Jin-Itering) [inline grant/projects/evolutionland.md:15]: 1000+ DAO?

Comment (WoeOm) [reply grant/projects/evolutionland.md:15]: DAU(Daily Active User)

File: grant/projects/evolutionland.md [hunk 2]
```diff
@@ -0,0 +1,118 @@
...
+- Furnance System - First NFT + FT Gamefi gameplay
+- Buy, sell, destroy and send functions
+- Apostle breeding system
+- Profession system
+
+## **Future Development Roadmap**
+
+The following milestones are not included in the collaboration grant assessment and is only here to communicate Evolution Land team's vision and direction.
+
+### Evolution Land **1.0**
```
Comment (Jin-Itering) [inline grant/projects/evolutionland.md:68]: Why is 1.0 scope the same as beta version?

File: grant/projects/evolutionland.md [hunk 3]
```diff
@@ -0,0 +1,118 @@
...
+
+Total funding amount: 6,000 USDT. Payment to be made in two instalments to the value of $ 3,000 USD.
+
+The first installment is paid in RING to the value of $3,000 USD. The ratio of RING to determined by average rate of the prior week, set by the time of accepting the pull request. Price ratio to be generated by coinmarket.com or coingecko.com - if a automatited 7 day average is obtainable. The 7 day high + 7 day low / 2 is also acceptable - allowing for 8 points behind the decimal at each 7 day price point.
+
+The second payment is paid in USDT to the value of $3,000 USD on completion of the acceptance criteria judged by Darwinia.
+
+## **Additional Information**
+
+To trial our pre-beta product please go to: Evolution Land. You will need the following:
```
Comment (Jin-Itering) [inline grant/projects/evolutionland.md:100]: Columbus is redirected to https://www.reddit.com/r/EvolutionLand/?

Comment (WoeOm) [reply grant/projects/evolutionland.md:100]: Columbus land is not yet available for production version, https://www.evolution.land.l2me.com/land/3 is a beta version.

File: grant/projects/evolutionland.md (hunk 2 above)
Comment (gravitiny) [reply grant/projects/evolutionland.md:68]: beta version is only for test, 1.0 scope would be the formal one

File: grant/projects/evolutionland.md [hunk 4]
```diff
@@ -0,0 +1,119 @@
...
+
+### Evolution Land **4.0:**
+
+- Building system
+
+## **Colabriation Funding**
+
+Total funding amount: 6,000 USDT. Payment to be made in two instalments to the value of $ 3,000 USD.
+
+The first installment is paid in RING to the value of $3,000 USD. The ratio of RING to determined by average rate of the prior week, set by the time of accepting the pull request. Price ratio to be generated by coinmarket.com or coingecko.com - if a automatited 7 day average is obtainable. The 7 day high + 7 day low / 2 is also acceptable - allowing for 8 points behind the decimal at each 7 day price point.
```
Comment (Jin-Itering) [inline grant/projects/evolutionland.md:94]: It is better to specify the accurate payment date

**Comment by gravitiny** [comment]
updated

File: grant/projects/evolutionland.md [hunk 5]
```diff
@@ -0,0 +1,123 @@
...
+Evolution Land is the first **Metaverse+Gamefi+cross-chain game** with each continent built on different blockchain networks, such as the first Atlantis continent. The game will have up to 26 continents, each of which is deployed on a different public chain. Four continents have been built so far and have been deployed on Ethereum (Atlantis Continent), Tron (Byzantine), Columbus (Crab Network), Dawning (Heco) and Eden(Polygon).
+
+Evolution Land integrates various gameplays, including management, collection, education, gene, battle, auction, and game, which will continuously be evolving.
+
+**One Dapp One Community Metaverse**
+10+ projects have announced plans to build communities in Evolution Land to present project information and give back rewards to the community fans.
+
+### **Why is this project is good for the Crab ecosystem:**
+
+1. Evolution Land is the first build GameFi project on Crab Smart Chain. Evolution Land has been built on four network - Ethereum, Tron, HECO, and Polygon. Each of them have at least 200,000+ $RING TVL. It will definately bring more TVL on Crab Smart Chain.
```
Comment (Jin-Itering) [inline grant/projects/evolutionland.md:14]: four networks

Comment (Jin-Itering) [reply grant/projects/evolutionland.md:14]: each of them **has**

File: grant/projects/evolutionland.md [hunk 6]
```diff
@@ -0,0 +1,123 @@
...
+2. Evolution Land have 1000+ DAU from HECO, Ethereum, Tron, and Polygon. And will build on BSC and Moonriver in the future. It could introduce more users and assets from other chains.
```
Comment (Jin-Itering) [inline grant/projects/evolutionland.md:15]: have -> has

File: grant/projects/evolutionland.md [hunk 7]
```diff
@@ -0,0 +1,123 @@
...
+3. Multi-chain and cross-chain is an important gameplay on Evolution Land. This point is similar to the Darwinia Bridge.
+
+### **How you will integrate this project into Crab:**
+
+Evolution Land will take the advantage of EVM module on Crab Smart Chain. We will deploy our contract(ERC20, ERC721, ERC1155) on CSC. We alreay deployed our alpha version for testing. Once our beta is released we will enable the community and Darwinia to start minting and playing.
```
Comment (Jin-Itering) [inline grant/projects/evolutionland.md:20]: alreay -> already

File: grant/projects/evolutionland.md [hunk 8]
```diff
@@ -0,0 +1,123 @@
...
+
+### **Proposal**
+
+1. Build and deploy Evolution Land on Crab Network: a **Metaverse+Gamefi+cross-chain game**. 
+2. Start marketing events on Crab continent - Columbus.
+
+Evolution Land will provide the following to Darwinia and Crab community:
+
+- Latest version including PVE, equipment, and profession gameplay
+- Continously update on Columbus continent
```
Comment (Jin-Itering) [inline grant/projects/evolutionland.md:48]: Continuously

File: grant/projects/evolutionland.md [hunk 9]
```diff
@@ -0,0 +1,123 @@
...
+- 70% of profit will return to users
+
+## Evolution Land **Beta Version(Only for test):**
+
+The beta version of Evolution Land is what is applying for CRAB grant program ( [Preview Link](https://www.evolution.land.l2me.com/land/3)
+), and the delivery scope has:
+
+- Basic mining gameplay including drills and apostles mining functions
+- Apostle Adventure - PVE gameplay
+- Furnance System - First NFT + FT Gamefi gameplay
```
Comment (Jin-Itering) [inline grant/projects/evolutionland.md:59]: Furnace?

File: grant/projects/evolutionland.md [hunk 10]
```diff
@@ -0,0 +1,123 @@
...
+## **Additional Information**
+
+To trial our pre-beta product please go to: Evolution Land. You will need the following:
+
+- Metamask or EVM related wallet
+- Crab for gas and xRING for purchase
+
+### **What work has been done so far?**
+
+Evolution Land's minimum viable Product has been made and is nearing its beta testing phase which is to be tested by the Darwinia community in end of Feb 2022.
```
Comment (Jin-Itering) [inline grant/projects/evolutionland.md:111]: In the end of

**Comment by 1022ranji** [comment]
Hi, I got some test tokens, and it displayed on my dashboard, but on the left corner on the top, there isn't a display of my balance.
image

**Review by Jin-Itering — CHANGES_REQUESTED** [review]
please fix the typos

**Comment by 1022ranji** [comment]
Function test 1:
Basic mining gameplay including drills and apostles mining functions
- [ ] Buy apostles
- [ ] Buy drills
- [ ] Buy land
- [ ] Put apostles and drills on the land that I bought before

**Comment by 1022ranji** [comment]
There might isn't an issue:
Before I buy the land/apostle/drill, there is a step called "approve", and this step cost a long time.

**Comment by gravitiny** [comment]
> There might isn't an issue: Before I buy the land/apostle/drill, there is a step called "approve", and this step cost a long time.

https://github.com/evolutionlandorg/evo-static/issues/776

**Comment by gravitiny** [comment]
> Hi, I got some test tokens, and it displayed on my dashboard, but on the left corner on the top, there isn't a display of my balance. image

https://github.com/evolutionlandorg/evo-static/issues/776

**Comment by gravitiny** [comment]
> There might isn't an issue: Before I buy the land/apostle/drill, there is a step called "approve", and this step cost a long time.

https://github.com/evolutionlandorg/evo-static/issues/777

**Review by Jin-Itering — APPROVED** [review]
LGTM

**Comment by WoeOm** [comment]
> There might isn't an issue: Before I buy the land/apostle/drill, there is a step called "approve", and this step cost a long time.

Is there a more detailed description about `this step cost a long time`?  Is it possible to provide transaction hash so that we can confirm the problem

**Comment by WoeOm** [comment]
> Hi, I got some test tokens, and it displayed on my dashboard, but on the left corner on the top, there isn't a display of my balance. image

A ui bug, which has been fixed.

**Comment by 1022ranji** [comment]
After I claimed the land, there also displayed like this, and after several times refresh, it'll be normal:
tx: https://crab.subview.xyz/tx/0xb9cb1177086738129b6a501033272356c53bdbfa855a04b168309978fdcc8717

image

**Comment by 1022ranji** [comment]
Function test 1 done.
Function test 2:
- [ ] Apostle Adventure - PVE gameplay

**Comment by gravitiny** [comment]
> After I claimed the land, there also displayed like this, and after several times refresh, it'll be normal: tx: https://crab.subview.xyz/tx/0xb9cb1177086738129b6a501033272356c53bdbfa855a04b168309978fdcc8717
> 
> image

We are immigrating staging version now. You could try it tomorrow.

**Comment by 1022ranji** [comment]
Function test 2 done.
Function test 3:
Finance System - First NFT + FT Gamefi gameplay
- [ ] Buy resource
- [ ] Generate LP
- [ ] Farm function
- [ ] Claim staking rewards
- [ ] Forge/Enhance,/dismantle equipment and tools

**Review by HackFisher — APPROVED** [review]

**Comment by 1022ranji** [comment]
- [ ] Buy, sell, and send functions
- [ ] Apostle breeding system
- [ ] Profession system
All Function test is done.

**Pull request merged**